backend/routes_cycle_count.py

```
@@ -860,12 +860,32 @@ def get_count_discrepancies():
             return jsonify({'error': 'An error occurred while fetching count discrepancies'}), 500
 
+    # Get a specific count result
+    @app.route('/api/cycle-counts/results/<int:result_id>', methods=['GET'])
+    @tool_manager_required
+    def get_count_result(result_id):
+        try:
+            # Get result
+            result = CycleCountResult.query.get_or_404(result_id)
+
+            # Get the associated item
+            item = CycleCountItem.query.get(result.item_id)
```

Ruff (0.11.9)

Local variable `item` is assigned to but never used

Remove assignment to unused variable `item`

(F841)